schemas/fingerprint-related-visitors-api.yaml

```diff
@@ -0,0 +1,93 @@
+openapi: 3.0.3
+info:
+  title: Related Visitors API
+  summary: |
+    Use this API to unify and personalize the experience for your anonymous users who visit your website from regular and in-app browsers. 
+  version: '1.0'
+  contact:
+    name: Fingerprint Support
+    email: support@fingerprint.com
+tags:
+  - name: Related Visitors
+servers:
+  - url: https://api.fpjs.io
+    description: Recommended if your customer base is primarily in the Americas.
+  - url: https://eu.api.fpjs.io
+    description: Recommended if your customer base is primarily in Europe.
+  - url: https://ap.api.fpjs.io
+    description: Recommended if your customer base is primarily in the APAC region.
+securityDefinitions:
+  APIKeyHeader:
+    type: apiKey
+    in: header
+    name: Auth-API-Key
+  APIKeyQueryParam:
+    type: apiKey
+    in: query
+    name: api_key
+security:
+  - APIKeyHeader: []
+  - APIKeyQueryParam: []
+paths:
+  /related-visitors?visitor_id={visitor_id}:
+    get:
+      tags:
+        - Related Visitors
+      operationId: 'getRelatedVisitors'
+      summary: Get Related Visitors
+      description: |
+        This API will enable you to find if one or more web and in-app browser sessions originated from the same mobile device.
+        When requested, this API will search identification events within the past 6 months to find the visitor IDs that belong to the same mobile device as the given visitor ID.
+        Please visit the [Overview](https://dev.fingerprint.com/reference/related-visitors-api) page to learn more about this API.
+      parameters:
+        - name: visitor_id
+          description: The [visitor ID](https://dev.fingerprint.com/docs/js-agent#visitorid) for which you want to find the other visitor IDs that originated from the same mobile device.
+          in: query
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK.
+        '400':
+          description: Bad request. The visitor ID parameter is missing or in the wrong format.
+          content:
+            application/json:
+              schema:
+                $ref: 'fingerprint-server-api.yaml#/components/schemas/ErrorCommon400Response'
+              examples:
+                400-invalid-visitor-id:
+                  summary: Error response when the visitor ID is incorrectly formatted or is missing
+                  externalValue: '/examples/delete_visits_400_error.json'
+        '403':
+          description: Forbidden. Access to this API is denied.
+          content:
+            application/json:
+              schema:
+                $ref: 'fingerprint-server-api.yaml#/components/schemas/ErrorCommon403Response'
```

Comment:
Suggestion: maybe it would make sense to extract shared response types to separate `yaml` file?

Reply:
The bigger problem is that these cross-file references don't really work for our use case, we can only upload a single OpenAPI file to Readme at a time. I need to create another transformer which takes the schema from the other file and appends it to this one.

Reply:
So, how do we go about this suggestion? Do I create a new file or duplicate the schema for error responses in this file? Or something else?

Reply:
@JuroUhlar , @TheUnderScorer - Here is what I have done:

> The bigger problem is that these cross-file references don't really work for our use case

I **did not** extract shared responses to a separate file. Instead, I made file `fingerprint-related-visitors-api.yaml `a self-sufficient unit that contains all the response schemas.

Reply:
>  I made file fingerprint-related-visitors-api.yaml a self-sufficient unit that contains all the response schemas.

Thank you, but this was not necessary, I am sorry for the misunderstanding. We can handle this automatically:

>  I need to create another transformer that takes the schema from the other file and appends it to this one.

We already apply a bunch of transformations to the schema files before publishing, this will just be one more. It means each common response will only be defined in one place like you originally intended, making maintenance easier :) 

I will apply this change, no worries

Reply:
"The bigger problem" misled me to think that this is a super-hard task to do. But, thank you for making the changes.

Reply:
Clumsy wording on my part, apologies 😿

Reply:
On the original question, I would use the main yaml file as a source of common schemas for now. I don't see any benefit in having 2 files with external references that need to resolved through scripting instead of 1.